src/asm/fstack.c

```diff
@@ -342,6 +373,38 @@ void fstk_RunInclude(char const *path)
 	contextStack->uniqueID = macro_UndefUniqueID();
 }
 
+// Similar to `fstk_RunInclude`, but not subject to `-MG` or to `-i` search paths, and
```

Comment:
gcc's `-include` option uses the same search paths as regular `#include`. <https://gcc.gnu.org/onlinedocs/gcc/Preprocessor-Options.html> Shouldn't rgbasm behave the same way, handling `-i` search paths even if it doesn't handle `-MG`? I can imagine this being useful if you always pass `rgbasm --includefile charmap.asm` but vary the `--include` path depending on i18n.

Reply:
I think `-MG` support would make sense too. They *are* dependencies, and having `MG` support them means not having to list them explicitly in two places (the dependency list and the `--includefile` list). Again, it's how gcc works:

```
$ cat hello.h
#pragma once
int hello(void);
$ cat hello.c
#include "hello.h"
int hello() { return 42; }
$ gcc -M -MG hello.c
hello.o: hello.c /usr/include/stdc-predef.h hello.h
$ gcc -M -MG -include foo.h hello.c
hello.o: hello.c /usr/include/stdc-predef.h foo.h hello.h
```